setup.py

```diff
@@ -22,7 +22,7 @@
     python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
     install_requires=[
         "Django>=1.11",
-        "redis>=2.10.0",
+        "redis==2.10.6",
```

Comment:
I think a range like `<3.0.0` would be more accurate and usable.

Reply:
LGTM 👍 
Done ✅